### PR TITLE
[4.x] Display Guard Name(s) on Auth Panel

### PR DESCRIPTION
**Pull request by moisish**
This is the updated PR of what Taylor requested on a previous PR here https://github.com/laravel/telescope/pull/1308 to display the guard name(s) on the Authenticated Panel.

### CODE REVIEW

**Comment by taylorotwell** [comment]
What will be displayed for old entires that do not have the guard information stored?

**Comment by moisish** [comment]
Will display N/A

image

**Comment by taylorotwell** [comment]
Just hide the Guard label and value entirely if it is not present please. Please mark as ready for review again when done.

File: src/IncomingEntry.php [hunk 1]
```diff
@@ -319,4 +320,24 @@ public function toArray()
...
+                return config("auth.providers.{$guard['provider']}.model");
+            })
+            ->filter(fn ($model) => get_class($this->user) === $model)
+            ->keys();
+
+        if ($guards->isNotEmpty()) {
+            return $guards->all();
+        }
+
+        return [config('auth.defaults.guard')];
```
Comment (taylorotwell) [inline src/IncomingEntry.php:341]: Eh, on second look I don't think is necessarily accurate. You could have multiple guards using the same model? That doesn't mean it's the guard that authenticated this request.

**Pull request closed without merging**